### PR TITLE
V1 2 3

### PR DESCRIPTION
**Pull request by stockholmux**
### Description
- Adds artifact and version metadata for v1.2.3
- Updates blog post 1.2.3
### Issues Resolved

opensearch-project/OpenSearch#1734

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by elfisher — DISMISSED** [review]
LGTM!

File: _versions/2021-12-22-opensearch-1.2.3.markdown [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    artifact: opensearch-dashboards
+    version: 1.2.0
+  -
+    role: command-line-tools
+    artifact: opensearch-cli
+    version: 1.1.0
+  -
+    role: ingest
+    artifact: logstash-oss-with-opensearch-output-plugin
+    version: 7.16.1
```
Comment (VijayanB) [inline _versions/2021-12-22-opensearch-1.2.3.markdown:24]:
```suggestion
    version: 7.16.2
```

**Review by VijayanB — CHANGES_REQUESTED** [review]
new logstash image is released

**Comment by elfisher** [comment]
Closing and replacing with https://github.com/opensearch-project/project-website/pull/602/files so that DCO can be properly signed on the Logstash version update commit.

**Pull request closed without merging**